pkgs/applications/networking/cluster/terraform-providers/update-provider

```diff
@@ -47,31 +96,41 @@ prefetch_github() {
   nix-prefetch-url --unpack "https://github.com/$owner/$repo/archive/$rev.tar.gz"
 }
 
+version="$(jq -r '.version' <<< "$registry_response")"
+if [[ "$old_version" = "$version" && "$force" != 1 ]]; then
+  echo "$provider_name is already at version $version"
+  exit
```

Comment:
```suggestion
  exit 1
```

Reply:
It's not an error if the provider is already up to date